src/tests/ftest/util/ior_utils.py

```diff
@@ -299,7 +300,8 @@ def get_launch_command(self, manager, attach_info, processes, hostfile):
         return "{}{} {} {}".format(
             exports, manager, " ".join(args), self.__str__())
 
-    def run(self, manager, attach_info, processes, hostfile, display=True):
+    def run(self, manager, attach_info, processes, hostfile, client_log,
```

Comment:
To avoid breaking other calls, shouldn't this be optional (`client_log=None`).  Also missing a docstring